include/z64.h

```diff
@@ -947,7 +947,7 @@ typedef struct GlobalContext {
     /* 0x11E18 */ s16 unk_11E18;
     /* 0x11E1A */ s16 nextEntranceIndex;
     /* 0x11E1C */ char unk_11E1C[0x40];
-    /* 0x11E5C */ s8 unk_11E5C;
+    /* 0x11E5C */ s8 shootingGalleryAmmo;
```

Comment:
This isn't quite the ammo count so the name could be confusing. Might be a good idea to rename bombchuBowlingAmmo to something more generic as well, with comments to explain what kind of information they can contain.

Reply:
just go with shootingGalleryState and bombchuBowlingState, perhaps?

Reply:
does it represent a state? im having a hard time understanding all of the things its used for in the code

Reply:
shooting gallery ammo here does a number of things. during play, it's equal to 1 + ammo, but at the end of the game it has some negative values it uses for signaling. It also acts as a timer when you shoot your last arrow to give it a chance to hit a rupee before ending the game. So...yeah, lots of different things it does.

On the other hand, bombchuBowlingAmmo is probably accurate enough. It gets set to -1 to signal the end of the game, but that's about it.

Reply:
it being a timer and stuff too makes it seem even more general than a state to me. `shootingGalleryVar` or something idk lol
either way we're not gonna be able to make a word to describe that and will need to delegate it to a comment